testsuite/cmake/CMakeLists.txt

```diff
@@ -12,29 +12,16 @@ function(CMAKE_TEST)
 endfunction(CMAKE_TEST)
 
 if(WITH_PYTHON AND PYTHONINTERP_FOUND)
-  set(TESTING_PYTHON TRUE)
   # obtain Python*_SITEARCH without find_package(Python*)
   execute_process(
     COMMAND "${PYTHON_EXECUTABLE}" -c "from distutils import sysconfig as sc;print(sc.get_python_lib(prefix='', plat_specific=True, standard_lib=False))"
     OUTPUT_VARIABLE Python_SITEARCH
     OUTPUT_STRIP_TRAILING_WHITESPACE)
-else()
-  set(TESTING_PYTHON FALSE)
-endif()
-set(DESTDIR "${CMAKE_CURRENT_BINARY_DIR}/install")
 
-add_custom_target(setup_install)
-add_custom_target(setup_install_destdir COMMAND make install DESTDIR="${DESTDIR}" WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
-add_dependencies(setup_install setup_install_destdir)
-cmake_test(FILE test_install_destdir.sh DEPENDENCIES BashUnitTests.sh test_install.sh)
-if(CMAKE_INSTALL_PREFIX STREQUAL "/tmp/espresso-unit-tests")
-  add_custom_target(setup_install_prefix COMMAND make install WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
-  add_dependencies(setup_install setup_install_prefix)
-  cmake_test(FILE test_install_prefix.sh DEPENDENCIES BashUnitTests.sh)
+  cmake_test(FILE test_install.sh DEPENDENCIES BashUnitTests.sh)
```

Comment:
It would be useful to check if any `.py` files got installed outside of `@PYTHON_DIR@`.

Reply:
Do you mean if a `.py` file was installed by mistake somewhere in `/tmp/espresso-unit-tests` outside the `espressomd` folder?

Reply:
Exactly!

Reply:
Done.